packages/react/src/errorboundary.tsx

```diff
@@ -66,6 +66,25 @@ const INITIAL_STATE = {
   eventId: null,
 };
 
+function setCause(error: Error & { cause?: Error }, cause: Error): void {
+  const seenErrors = new Map<Error, boolean>();
```

Comment:
Should we make this a WeakMap to avoid GC issues? Probably doesn't matter but doesn't hurt I guess..?